content/rs/security/certificates/updating-certificates.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Update certificates
+linkTitle: Update certificates
+description: Update certificates in a Redis Enterprise cluster.
+weight: 20
+alwaysopen: false
+categories: ["RS"]
+aliases: ["/rs/administering/cluster-operations/updating-certificates",
+"/rs/administering/cluster-operations/updating-certificates/",
+"/rs/administering/cluster-operations/updating-certificates.md"]
+---
+
+{{<warning>}}
+When you update the certificates, the new certificate replaces the same certificates on all nodes in the cluster.
+{{</warning>}}
+
+## How to update certificates
+
+You can use either the rladmin command-line interface (CLI) or the REST API to update certificates.
+
+The new certificates are used the next time the clients connect to the database.
+
+When you upgrade Redis Enterprise Software, the upgrade process copies the certificates that are on the first upgraded node to all of the nodes in the cluster.
+
+### Use the CLI
+
+To replace certificates using the rladmin CLI, run:
+
+```sh
+ rladmin cluster certificate set <cert-name> certificate_file <cert-file-name>.pem key_file <key-file-name>.pem
+```
+
+Where:
+
+- cert-name - The name of the certificate you want to replace:
+  - For the admin console: `cm`
+  - For the REST API: `api`
+  - For the database endpoint: `proxy`
+  - For the syncer: `syncer`
+  - For the metrics exporter: `metrics_exporter`
+- cert-file-name - The name of your certificate file
+- key-file-name - The name of your key file
+
+For example, to replace the cm certificate with the private key "key.pem" and the certificate file "cluster.pem":
+
+```sh
+rladmin cluster certificate set cm certificate_file cluster.pem key_file key.pem
+```
+
+### Use the REST API
+
+To replace a certificate using the REST API, run:
+
+```sh
+curl -k -X PUT -u "<username>:<password>" -H "Content-Type: application/json" -d '{ "name": "<cert_name>", "key": "<key>", "certificate": "<cert>" }' https://<cluster_address>:9443/v1/cluster/update_cert
+```
+
+Where:
+
+- cert_name - The name of the certificate to replace:
+  - For the admin console: `cm`
+  - For the REST API: `api`
+  - For the database endpoint: `proxy`
+  - For the syncer: `syncer`
+  - For the metrics exporter: `metrics_exporter`
+- key - The contents of the \*\_key.pem file
```

Comment:
```suggestion
- `<key>` - The contents of the \*\_key.pem file
```
This will help differentiate so they don't replace the wrong "key" in the command.